### PR TITLE
add native api view test

### PR DESCRIPTION
**Pull request by gmlwhdtjd**
- For comparing the performance of Native api and SwiftLayout
- There is a problem that the snapshot of NativeApiView is not normal

### CODE REVIEW

File: Tests/SwiftLayoutTests/PerformanceTests.swift [hunk 1]
```diff
@@ -40,15 +45,30 @@ class PerformanceTests: XCTestCase {
...
     func testPerformanceLayoutable() throws {
         let metrics: [XCTMetric] = [XCTCPUMetric(), XCTMemoryMetric()]
-
         self.measure(metrics: metrics) {
             _ = SwiftLayoutView(frame: .init(x: 0, y: 0, width: 375, height: 667))
         }
     }
     
     func testPerformanceInterfaceBuilderAndLayout() throws {
+        let metrics: [XCTMetric] = [XCTCPUMetric(), XCTMemoryMetric()]
```
Comment (oozoofrog) [inline Tests/SwiftLayoutTests/PerformanceTests.swift:63]: mismatching interface and native 😉

**Review by oozoofrog — CHANGES_REQUESTED** [review]

**Comment by oozoofrog** [comment]
@gmlwhdtjd one idea came up by this PR. it's debug mode Debug mode allows you to print the layout's structure and constraint attributes to the log console when updating.

File: Tests/SwiftLayoutTests/PerformanceTests.swift (hunk 1 above)
Comment (gmlwhdtjd) [reply Tests/SwiftLayoutTests/PerformanceTests.swift]: haha! Oh my, did I make this mistake? 😅
I corrected it.

**Review by oozoofrog — APPROVED** [review]

**Comment by oozoofrog** [comment]
@gmlwhdtjd its wonderful works.

**Comment by gmlwhdtjd** [comment]
> one idea came up by this PR. it's debug mode. debug mode allows layoutable to print the layout's structure and constraint attributes to the log console when updating.

@oozoofrog 
`layout.debugPrint()` You mean like this?

**Comment by oozoofrog** [comment]
> > one idea came up by this PR. it's debug mode. debug mode allows layoutable to print the layout's structure and constraint attributes to the log console when updating.
> 
> @oozoofrog `layout.debugPrint()` You mean like this?

correct!

**Pull request merged**